### PR TITLE
[IMP] Purchase: update reporting

### PR DESCRIPTION
**Pull request by hojo-odoo**
This PR rewrites the doc detailing how to use the Purchase report.

### CODE REVIEW

**Comment by hojo-odoo** [comment]
Hey @jero-odoo,

This PR is ready for the first round of review.

Thanks,
John

**Review by jero-odoo — APPROVED** [review]
Hey @hojo-odoo nice work on this, I think the use cases are really well done. This is ready to go to the next round!

**Comment by hojo-odoo** [comment]
Hey @ksc-odoo,

This is ready for final review.

Thanks,
John

File: content/applications/inventory_and_mrp/purchase/advanced/analyze.rst [hunk 1]
```diff
@@ -1,122 +1,142 @@
...
-For bar and line charts, the selected measure is your y-axis, and the first group you select is used
-to create the x-axis. Adding more group adds additional lines (line chart) or transforms your bar
-chart into a stacked bar chart. For pie charts, the more groups you select, the more slices are
-displayed.
+   Finally, David clicks the :icon:`fa-minus-square-o` :guilabel:`Total` button above the rows on
+   the pivot table, and selects the :guilabel:`Product` option.
 
-Explore your data
-~~~~~~~~~~~~~~~~~
+   With all of these options configured, the Purchase report shows a pivot table comparing purchase
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/purchase/advanced/analyze.rst:128]:
```suggestion
   With all of these options configured, the :guilabel:`Purchase Analysis` report shows a pivot table comparing purchase
```

File: content/applications/inventory_and_mrp/purchase/advanced/analyze.rst [hunk 2]
```diff
@@ -1,122 +1,142 @@
...
+   for the taxed total spent. These columns are further broken down into three smaller columns: the
+   amount spent in May, the amount spent in June, and the variation between the two months,
+   represented as a percentage.
 
-.. image:: analyze/analyze-pivot-view.png
-   :align: center
-   :alt: Pivot table view in Odoo Purchase
+   On the left side of the pivot table, one row is shown for each product purchased from Bike
+   Friends during June. Using this report, David is able to see that Bike Haus has spent much less
+   money overall on products purchased from Bike Friends, compared to the previous month.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/purchase/advanced/analyze.rst:138]:
```suggestion
   On the left side of the pivot table, one row is shown for each product purchased from *Bike
   Friends* during June. Using this report, David is able to see that *Bike Haus* has spent much less
   money on products purchased from *Bike Friends*, compared to the previous month.
```

**Review by ksc-odoo — APPROVED** [review]
Hey @hojo-odoo -- just finished my Final Review of this PR. Great work. Once you address and implement the changes/modifications left throughout the PR, you can feel free to move it to the next stage. Thanks! 👍

**Comment by hojo-odoo** [comment]
Hey @samueljlieber and @StraubCreative,

This PR is ready for tech review.

Thanks,
John

File: content/applications/inventory_and_mrp/purchase/advanced/analyze.rst [hunk 3]
```diff
@@ -1,122 +1,146 @@
-=========================================
-Analyze the performance of your purchases
-=========================================
+=================
+Purchase Analysis
+=================
```
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/purchase/advanced/analyze.rst:3]: Please either add "report" or use full sentence case in the H1 :)
```suggestion
========================
Purchase Analysis report
========================
```

```suggestion
=================
Purchase analysis
=================
```

File: content/applications/inventory_and_mrp/purchase/advanced/analyze.rst [hunk 4]
```diff
@@ -1,122 +1,146 @@
...
+   Only one measure can be selected at a time when one of the :icon:`fa-area-chart`
+   :guilabel:`(graph view)` options is enabled. However, multiple measures, and varying group-by
+   criteria (on the x and y axes), can be selected when using the :icon:`oi-view-pivot`
+   :guilabel:`(pivot table)`.
 
-Odoo offers several default filters you can use and combine by clicking on *Filters*. When you
-select one or several filters, Odoo looks for all the orders that match at least one of the filters
-you have chosen and populates the chart, key metrics, and pivot table with the data. The
-pre-configured filters are:
+.. _purchase/advanced/analyze/example:
```
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/purchase/advanced/analyze.rst:56]: In custom anchors, only the parent folder is optional to include, and with it being named "advanced", I think we can omit it, and just use a little more context :)
```suggestion
.. _purchase/purchase-analysis-example:
```

File: content/applications/inventory_and_mrp/purchase/advanced/analyze.rst [hunk 5]
```diff
@@ -1,122 +1,146 @@
...
+Another use for the :guilabel:`Purchase Analysis` report is to compare key statistics about |POs|
+for two different time periods, for a specific vendor. By doing so, it is easy to understand how
+purchases from the vendor have increased or decreased.
 
-Before selecting what you want to measure, you need to decide whether you prefer using the graph
-view or the pivot view. By default, the dashboard presents both views. However, the measures you
-select won't be applied to both views. You can access each view separately by clicking on the icons
-at the top right of the dashboard.
+.. example::
+   Following the :ref:`previous example <purchase/advanced/analyze/example>`, it has been one month
```
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/purchase/advanced/analyze.rst:108]:
```suggestion
   Following the :ref:`previous example <purchase/purchase-analysis-example>`, it has been one month
```

File: content/applications/inventory_and_mrp/purchase/advanced/analyze.rst [hunk 6]
```diff
@@ -1,122 +1,146 @@
...
+   On the left side of the pivot table, one row is shown for each product purchased from Bike
+   Friends during June. Using this report, David is able to see that Bike Haus has spent much less
+   money on products purchased from Bike Friends, compared to the previous month.
 
-.. tip::
-   You can insert your pivot table's data directly in the Spreadsheet app or export it as an Excel
-   file.
+   .. image:: analyze/comparison.png
+      :align: center
+      :alt: The Purchase report, comparing the amount spent at a vendor,
```
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/purchase/advanced/analyze.rst:146]: Update to a period :)
```suggestion
      :alt: The Purchase report, comparing the amount spent at a vendor.
```

**Review by samueljlieber — APPROVED** [review]
Hi @hojo-odoo, nice work on this overhaul of the Purchase Analysis report doc! This is a great improvement, and I like how the use cases build off one another, cool concept!
Approving with a couple quick suggestions and fixes, thank you for your work!
...
@robodoo delegate=hojo-odoo

**Comment by hojo-odoo** [comment]
@robodoo r+

**Pull request closed without merging**